client/signup/config/step-components.js

```diff
@@ -1,193 +1,78 @@
-	'site-or-domain': SiteOrDomainComponent,
-	'site-picker': SitePicker,
-	'site-style': SiteStyleComponent,
-	'site-title': SiteTitleComponent,
-	'site-topic': SiteTopicComponent,
-	'site-type': SiteTypeComponent,
-	survey: SurveyStepComponent,
-	'survey-user': UserSignupComponent,
-	test:
-		process.env.NODE_ENV === 'development'
-			? require( 'signup/steps/test-step' ).default
-			: undefined,
-	themes: ThemeSelectionComponent,
-	'website-themes': ThemeSelectionComponent,
-	'blog-themes': ThemeSelectionComponent,
-	'themes-site-selected': ThemeSelectionComponent,
-	user: UserSignupComponent,
-	'oauth2-user': UserSignupComponent,
-	'oauth2-name': UserSignupComponent,
-	displayname: UserSignupComponent,
-	'reader-landing': ReaderLandingStepComponent,
+	about: loadStep( 'about' ),
+	'clone-start': loadStep( 'clone-start' ),
+	'clone-destination': loadStep( 'clone-destination' ),
+	'clone-credentials': loadStep( 'clone-credentials' ),
+	'clone-point': loadStep( 'clone-point' ),
+	'clone-jetpack': loadStep( 'clone-jetpack' ),
+	'clone-ready': loadStep( 'clone-ready' ),
+	'clone-cloning': loadStep( 'clone-cloning' ),
+	'creds-confirm': loadStep( 'creds-confirm' ),
+	'creds-complete': loadStep( 'creds-complete' ),
+	'creds-permission': loadStep( 'creds-permission' ),
+	'design-type': loadStep( 'design-type' ),
+	'design-type-with-store-nux': loadStep( 'design-type-with-atomic-store' ),
+	domains: loadStep( 'domains' ),
+	'domains-store': loadStep( 'domains' ),
+	'domain-only': loadStep( 'domain' ),
+	'domains-theme-preselected': loadStep( 'domains' ),
+	'domains-launch': loadStep( 'domains' ),
+	'from-url': loadStep( 'import-url' ),
+	launch: loadStep( 'launch-site' ),
+	plans: loadStep( 'plans' ),
+	'plans-launch': loadStep( 'plans' ),
+	'plans-personal': loadStep( 'plans' ),
+	'plans-premium': loadStep( 'plans' ),
+	'plans-business': loadStep( 'plans' ),
+	'plans-store-nux': loadStep( 'plans-atomic-store' ),
+	'plans-site-selected': loadStep( 'plans-without-free' ),
+	site: loadStep( 'site' ),
+	'rebrand-cities-welcome': loadStep( 'rebrand-cities-welcome' ),
+	'rewind-migrate': loadStep( 'rewind-migrate' ),
+	'rewind-were-backing': loadStep( 'rewind-were-backing' ),
+	'rewind-add-creds': loadStep( 'rewind-add-creds' ),
+	'rewind-form-creds': loadStep( 'rewind-form-creds' ),
+	'site-information': loadStep( 'site-information' ),
+	'site-information-without-domains': loadStep( 'site-information' ),
+	'site-information-title': loadStep( 'site-information' ),
+	'site-information-address': loadStep( 'site-information' ),
+	'site-information-phone': loadStep( 'site-information' ),
+	'site-or-domain': loadStep( 'site-or-domain' ),
+	'site-picker': loadStep( 'site-picker' ),
+	'site-style': loadStep( 'site-style' ),
+	'site-title': loadStep( 'site-title' ),
+	'site-topic': loadStep( 'site-topic' ),
+	'site-type': loadStep( 'site-type' ),
+	survey: loadStep( 'survey' ),
+	'survey-user': loadStep( 'survey-user' ),
+	test: loadStep( 'test-step' ),
```

Comment:
This seems to change the existing behaviour. Perhaps we should preserve the condition and instead write:
```
process.env.NODE_ENV === 'development'	? loadStep( 'test-step' ) : undefined,
```
?

Reply:
My thinking was that the `test-step` is harmless and as it's async-loaded anyway, doesn't add any unneeded code. So I chose the simplest route.

Reply:
👍 If it doesn't affect what end users see, I'm ok with keeping it.